### DISCUSSION TITLE
Help Needed: Zigbee2MQTT Docker on Synology NAS - Error After Updating v1.35.3 to v1.39.1

### DISCUSSION THREAD

**Discussion by gino-nl** [Koenkk/zigbee2mqtt#23656] [General]
Hi everyone,

I’m running Zigbee2MQTT via **Docker** on my **Synology** NAS with **DSM 7.1**, using an **ITEAD SONOFF Zigbee 3.0 USB Dongle Plus V2 firmware 7.4.3.0 (BDongle-E)** as the coordinator device.
Everything was working perfectly until I updated Zigbee2MQTT from **v1.35.3 to v1.39.1**.

After the update, Docker failed to start, and I received the following warning:

`warning: zh:ezsp: Deprecated driver 'ezsp' currently in use, 'ember' will become the officially supported EmberZNet driver in next release. See https://github.com/Koenkk/zigbee2mqtt/discussions/21462`

Following this, I changed the adapter in my **configuration.yaml** from **ezsp** to **ember**, but now I’m encountering a new error:

`error: zh:ember:uart:ash: Failed to init port with error Error: Error: No such file or directory, cannot open /dev/ttyACM0`

I’ve tried troubleshooting, but I’m stuck and can’t figure out how to resolve this issue. Has anyone else encountered this problem? Any help or ideas would be greatly appreciated!

Thanks in advance!

![error](https://github.com/user-attachments/assets/4be0789f-9605-4b9f-ae9e-59d0b96bb358)

**Comment by chris-1243** [comment]
Could you provide your `docker-compose.yaml` and your `configuration.yaml`. It seems Z2M can not access your Dongle-E anymore

Do you use /dev/serial/by-id in your docker-compose.yaml?

**Reply by gino-nl** [reply]
Thanks for the reply. 
It seems that Synology does not support **/dev/serial/by-id**.

After doing more research and trial and error, I decided to rebuild the whole docker using this command:


`sudo docker run --name zigbee2mqtt -dit --device=/dev/ttyACM0 --network host --restart unless-stopped -v /home/admin/zigbee2mqtt/data:/app/data -v /run/udev:/run/udev:ro -e TZ=Europe/Amsterdam koenkk/zigbee2mqtt:latest
`

This solved my problem :-)
